mobify/platform-scaffold
# Dismissal config is via DefaultAsk prop now

**NOTE: Requires SDK release and version bump!**

The latest changes in the SDK introduce a breaking change to Push Messaging component configuration. The visits to wait upon dismissal prop is now configured on the `DefaultAsk` (or any other component implementing push messaging) via the `deferOnDismissal` prop

## Changes
- Remove `visitsToWaitIfDismissed={1}` from `<PushMessagingController ... />`
- Add `deferOnDismissal={1}` to `<DefaultAsk ... />`

## Todos
- [ ] SDK release and version bump

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- You should see the `DefaultAsk` component upon second page view
- Dismissing the ask or the system-ask should add a `{ default: 1 }` string to `pw-visitCountdowns` in localStorage
